skimage/metrics/set_metrics.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from ._set_metrics import hausdorff_distance_onesided
+
+
+def hausdorff_distance(a, b):
+    """
+    Calculate the Hausdorff distance [1]_ between two sets of points.
+
+    The Hausdorff distance is the maximum distance between any point on
+    ``a`` and its nearest point on ``b``, and vice-versa.
+
+    Parameters
+    ----------
+    a, b : ndarray, dtype=bool
+        Arrays where ``True`` represents a point that is included in a
+        set of points. Both arrays must have the same shape.
+
+    Returns
+    -------
+    distance : float
+        The Hausdorff distance between sets ``a`` and ``b``, using
+        Euclidian distance to calculate the distance between points in ``a``
+        and ``b``.
+
+    References
+    ----------
+    .. [1] http://en.wikipedia.org/wiki/Hausdorff_distance
+
+    Examples
+    --------
+    >>> points_a = (3, 0)
+    >>> points_b = (6, 0)
+    >>> shape = (7, 1)
+    >>> coords_a = np.zeros(shape, dtype=np.bool)
+    >>> coords_b = np.zeros(shape, dtype=np.bool)
+    >>> coords_a[points_a] = True
+    >>> coords_b[points_b] = True
+    >>> hausdorff_distance(coords_a, coords_b)
+    3.0
+
+    """
+    if a.dtype != np.bool or b.dtype != np.bool:
+        raise ValueError('Arrays must have dtype = \'bool\'')
+    if a.shape != b.shape:
+        raise ValueError('Array shapes must be identical')
```

Comment:
Personally, I would remove these checks.

Reply:
(certainly the first one!)

Reply:
Done